configs/model/padim.yaml

```
@@ -19,22 +21,29 @@ data:
     seed: 0
     create_validation_set: false
 
+tiling:
+  apply: false
+  tile_size: null
+  stride: null
+  remove_border_count: 0
+  mode: padding
+  tile_count: 4
+
 model:
   class_path: anomalib.models.Padim
   init_args:
-    input_size:
-      - 256 # Height
-      - 256 # Width
-    backbone: resnet18
-    pre_trained: true
-    layers:
-      - layer1
-      - layer2
-      - layer3
+    feature_extractor:
+      backbone: resnet18
+      pre_trained: true
+      layers:
+        - layer1
+        - layer2
+        - layer3
+    n_features: null
 
 post_processing:
-  normalization_method: min_max # <null, min_max, cdf>
-  threshold_method: adaptive # options: [adaptive, manual]
+  normalization_method: MIN_MAX
+  threshold_method: ADAPTIVE
```

As I was trying to use the CLI I realized it's harder to figure out where to find these possible values (say, from an error stack) because the Lightning layer makes it more indirect.
I would be nice (and more maintainable than the previous format) to leave comments like 

```suggestion 
  normalization_method: MIN_MAX  # see anomalib.post_processing.NormalizationMethod
  threshold_method: ADAPTIVE  # see anomalib.post_processing.ThresholdMethod
```

---

I specifically suggested `anomalib.post_processing.ThresholdMethod` instead of `anomalib.post_processing.post_process.ThresholdMethod` (the shortest submodule path rather than the most specific) in order to reduce the chance of having to change the path.